kitsune/dashboards/readouts.py

```diff
@@ -708,15 +717,28 @@ def _query_and_params(self, max):
         else:
             period = self.default_mode
 
+        ignore_categories = [str(ADMINISTRATION_CATEGORY),
+                             str(NAVIGATION_CATEGORY),
+                             str(HOW_TO_CONTRIBUTE_CATEGORY)]
+
         # Filter by product if specified.
         if self.product:
             extra_joins = PRODUCT_FILTER
             params = (self.locale, period, self.product.id,
                       settings.WIKI_DEFAULT_LANGUAGE)
+
+            has_forum = self.product.questions_locales.filter(locale=self.locale).exists()
         else:
             extra_joins = ''
             params = (self.locale, period, settings.WIKI_DEFAULT_LANGUAGE)
 
+        if not self.product or not has_forum:
+            ignore_categories.append(str(CANNED_RESPONSES_CATEGORY))
```

Comment:
If they dont have a product selected, we won't filter out canned responses. I know it's complicated otherwise, hmm.